tests/io_utils.go

```diff
@@ -45,32 +46,84 @@ func NodeNameWithHandler() string {
 	return node.ObjectMeta.Name
 }
 
-func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) error {
+func ExecuteCommandInVirtHandlerPod(nodeName string, args []string) (string, error) {
```

Comment:
Maybe we might name the first return argument, which would make clear what it returns.